lib/index.js

```diff
@@ -92,6 +92,29 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
         });
     }));
 
+  app.patch(
```

Comment:
A `PATCH` endpoint needs to accept a patch as post material. This should be changed to `POST`... or we should have a `PATCH` `account/meta` endpoint to handle this sort of thing.

Since the underlying API splits `status` up into its own API call I suppose it would be a bit cumbersome to handle that as a meta patch, so let's just change this to `api.post`.

Reply:
this is for setStatus? ok will do.